client/less/classic-modal.less

```diff
@@ -1,34 +0,0 @@
-.challenge-success-modal {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  height: 50vh;
-
-  .modal-header {
```

Comment:
Blame on the file says it hasn't been edited since it was merged. I had it mimic the production version